recipes/e3sm_to_cmip/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "e3sm_to_cmip" %}
+{% set version = "1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/E3SM-Project/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: a23d7440bab9950f6fef8a6e5fe60741e4e5be2a2421d37ca9a1739dab1deef1
+
+build:
+  skip: True  # [win or py<36]
```

Comment:
@sterlingbaldwin, just to make sure you agree that we don't want to support Windows or python <3.6

Reply:
Is there anything specific for ``unix`` platform which cannot be run on ``windows``?

Reply:
@sterlingbaldwin, you would know better than me.

Reply:
Actually, I think I can answer this.  We have some subprocess calls that have not been generalized for Windows (i.e. we call `ncremap` and not `ncreamp.exe`, as would be required in Windows).  We also don't foresee anyone wanting to use this package on Windows systems -- it really only makes sense on HPC.

Reply:
I dont think anything will break on py<3.6, but Im not going to be actively maintaining it for old python verions.

Reply:
If the feedstock supports py<36 but you're not maintaining it, the builds are bound to start breaking.  So I think we should keep that constraint.  What about Windows?

Reply:
> Actually, I think I can answer this. We have some subprocess calls that have not been generalized for Windows (i.e. we call `ncremap` and not `ncreamp.exe`, as would be required in Windows). We also don't foresee anyone wanting to use this package on Windows systems -- it really only makes sense on HPC.

That is fine then. It was just to double check! :)